authors/apps/authentication/tests_/test_endpoints.py

```diff
@@ -0,0 +1,62 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.urls import reverse
+import json
+from authors.apps.authentication.models import User
+# from authors.apps.authentication import
+
+
+class TestUserEndPoints(TestCase):
+    def setUp(self):
+        self.user_data = {
+            "user": {
+                "username": "Jacob",
+                "email": "jake@jake.jake",
+                "password": "jakejake"
+            }
+        }
+        self.verified_user = User.objects.create_user(
+            username='testuser', email='testemail@test.com', password='testpassword')
+
+        self.verified_user_login_credentials = {
+            "user": {
+                "email": "testemail@test.com",
+                "password": "testpassword"
+            }
+        }
+
+        self.invalid_user = {
+            "user": {
+                "email": "adddd@gmail.com",
+                "password": "Aaddadasas"
+
+            }
+        }
+
+        self.client = APIClient()
+
+    def test_user_regsitration(self):
```

Comment:
@MuhweziDeo Its a good practice to state what you are testing for e.g if you are testing for user registration success, the name of the function would be `test_user_registration_with_valid_data_succeeds`.

Reply:
alright @ja-odur